res/controllers/Denon-MC7000-scripts.js

```diff
@@ -1267,30 +1249,22 @@ MC7000.VelSampLED = function() {
             var sampNo = 0;
             if (MC7000.PADModeVelSamp[deckIdx]) {
                 if (MC7000.SamplerQty === 16) {
-                    sampNo = (deckIdx % 2) * 8 + velSampIdx;
-                    var inactiveVelSampDeck = (deckIdx >= 2) ? deckIdx - 2 : deckIdx + 2;
+                    sampNo = (deckIdx % 2) * 8 + velSampIdx; // use sampler 1-16 for deck 1 and deck 2 and sampler 1-16 for deck 3 and 4
```

Comment:
```suggestion
                    // use sampler 1-16 for deck 1 and deck 2 and sampler 1-16 for deck 3 and 4
                    sampNo = (deckIdx % 2) * 8 + velSampIdx;
```
Please shorten such long line

Reply:
done